awx_collection/plugins/modules/tower_job_template.py

```diff
@@ -55,8 +55,17 @@
     credential:
       description:
         - Name of the credential to use for the job template.
+        - Deprecated, mutually exclusive with 'credentials'.
       version_added: 2.7
       type: str
+    credentials:
+      description:
+        - List of credentials to use for the job template.
+        - Will not remove any existing credentials.
```

Comment:
Do want to consider the proposed batch attach/detatch .. or just consider that when it comes time?

Reply:
By that you mean https://github.com/ansible/awx/issues/5245. We should pull @beeankha into this conversation.

The time frame for that will still be a while down the road. I would like to get this in user's hands now. I think it makes sense to punt on removal until that lands.

Reply:
I agree that we shouldn't gate this change on the work @beeankha will be doing later for a bulk attach feature (though we could revisit this module at that point if we want to).